src/Lesson_1/Lesson_2.java

```diff
@@ -0,0 +1,72 @@
+package Lesson_1;
+
+public class Lesson_2 {
+
+    public static void main(String[] args) {
+        System.out.println(limit( 10,10));
+        PosNeg(7);
+        System.out.println(Otricat(-2));
+        StringNumber("яблоко",6);
+        System.out.println(CheckLeapYear());
+    }
+    //1.Задание ..
+    // Написать метод, принимающий на вход два целых числа и проверяющий,
+    // что их сумма лежит в пределах от 10 до 20 (включительно),
+    // если да – вернуть true, в противном случае – false.
+
+    static boolean limit(int a, int b) {
+        int sum = a + b;
+        return sum >= 10 && sum <= 20;
+    }
+    //2.Задание
+    // Написать метод, которому в качестве параметра передается целое число,
+    // метод должен напечатать в консоль, положительное ли число передали или отрицательное.
+    // Замечание: ноль считаем положительным числом.
+
+    static void PosNeg(int a) {
```

Comment:
методы именуются с прописной буквы
```suggestion
    static void posNeg(int a) {
```